bsp/boards/scum/leds.c

```diff
@@ -64,26 +78,38 @@ uint8_t leds_sync_isOn() {
 
 // 3 <N14>
 void    leds_debug_on() {
+#ifndef DEBUGPINS_IN_USE

```

Comment:
The leds.c file is not supposed to be committed. I will change it back when I finish the final port.